Code releases 05.20/d8e84299-d99b-40e5-a34d-791101435ee6.md

```diff
@@ -1,5 +1,39 @@
-Super Attributes in the Spryker Commerce OS are used to distinguish between the different Product Variants of an abstract product. Super Attributes define each Concrete Product and can consist of whichever distinguishing feature you wish to highlight, such as size or color. This information can either be manually managed or imported and processed automatically.
+The super attribute is the only way to tell Spryker, that a product contains multiple variants (or concrete products) to allow your customer to select the variant they are looking for.
+Spryker supports an unlimited amount of super attributes for a given abstract product. See below to understand how abstract and concrete products works with super attributes.
 
-Consider the example of a T-shirt, the Abstract Product, that is available in the sizes small, medium, and large, three different Product Variants. The Abstract Product "T-shirt" appears as a search result. On the product detail page the customer can select between the product variants small, medium, large and put it into the cart.
+## Hierarchy of products
 
-![image](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Product+Management/Product+Abstraction/product-abstraction.png){height="" width=""}
\ No newline at end of file
+In Spryker, you have 2 product levels:
+* Abstract
+* Concrete
+
+An abstract product can contain 1 or more concrete products. A concrete product is only linked to 1 abstract product.
+
+### Differences between the levels
+
+Abstract product is the level where all the information about the product are located such as title, description, images, prices, brand.
+Concrete product is the level where all divergent information compared to the abstract product are located such as size, color, images, prices. It is the level where the stock for the product is defined. It is the only level where super attributes must be defined.
+
+### Inheritance between levels
+
+Whenever Spryker must display some information about a concrete product, it will execute the following logic.
+* Does the information exists on the concrete product level?
+  * If yes, Spryker uses it.
+  * If no, Spryker will look if the information is present on the abstract product level.
```

Comment:
this is not complete true. When showing product concrete, we merge concrete and abstract products' information and use merged result as a source.

Reply:
I see what you mean. The logic is still valid (in theory) but the reality is different. I will rephrase this paragraph then